raft-tech/TANF-app
# Nginx DNS Issue: Kibana and ClamAV

## Summary of Changes
- Updated nginx config files to give certain hostnames a TTL.
  - Applies to the Kibana and the ClamAV hosts
Pull request closes #2839 

## How to Test
- This is deployed in the `raft` environment
  - Login to the env and verify you can see Kibana as an OFA Sys admin. If you still cant see the Kibana tab, follow the instructions [here](https://github.com/raft-tech/TANF-app/pull/2805).
  - Login to cloud.gov and go to the `raft` apps in the dashboard.
  - Restart the Kibana app
  - Verify during restart you cant see Kibana
  - Verify after the restart that you can see Kibana again (Before if you did this, you would have had to restart the frontend to be able to route to Kibana).

## Deliverables
_More details on how deliverables herein are assessed included [here](https://github.com/raft-tech/TANF-app/blob/develop/docs/How-We-Work/our-priorities-values-expectations.md#Deliverables)._

### [Deliverable 1: Accepted Features](https://github.com/raft-tech/TANF-app/blob/develop/docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-1-Accepted-Features)

Checklist of ACs:
- [ ] Determine if there is a way to automatically resolve the routing issues
- [x] Implement solution
- [ ] Reschedule Kibana and verify Frontend can still route to it without restaging it.

### [Deliverable 2: Tested Code](https://github.com/raft-tech/TANF-app/blob/develop/docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-2-Tested-Code)

+ Are all areas of code introduced in this PR meaningfully tested?
+ Are code coverage minimums met?

### [Deliverable 3: Properly Styled Code](https://github.com/raft-tech/TANF-app/blob/develop/docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-3-Properly-Styled-Code)


### [Deliverable 4: Accessible](https://github.com/raft-tech/TANF-app/blob/develop/docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-4-Accessibility)



### [Deliverable 5: Deployed](https://github.com/raft-tech/TANF-app/blob/develop/docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-5-Deployed)


### [Deliverable 6: Documented](https://github.com/raft-tech/TANF-app/blob/develop/docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-6-Code-documentation)


### [Deliverable 7: Secure](https://github.com/raft-tech/TANF-app/blob/develop/docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-7-Secure)


### [Deliverable 8: User Research](https://github.com/raft-tech/TANF-app/blob/develop/docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-8-User-Research)

Research product(s) clearly articulate(s):
